content/blogposts/2023/incident-draft.md

```diff
@@ -0,0 +1,127 @@
+
+Our internal control systems, including automated code analysis, failed to catch the access token being committed to the repository. 
+
+On August 28, 2023 _(2023-08-28 13:18:36 UTC)_, a user created a brand new Sourcegraph account. 
+
+On August 30, 2023 _(2023-08-30 06:47:59 UTC)_, using the leaked site-admin access token, this user elevated their account privileges to a site-admin and gained unauthorized access to the admin dashboard.
+
+The malicious user continued to probe the system by changing their access from a site-admin to regular user multiple times.
+
+The malicious user, or someone connected to them, created a proxy app allowing users to directly call Sourcegraph’s APIs and leverage the underlying LLM. Users were instructed to create free Sourcegraph.com accounts, generate access tokens, and then request the malicious user to greatly increase their rate limit.
+
+On August 30 _(2023-08-30 13:25:54 UTC)_, the Sourcegraph security team identified the malicious site-admin user, revoked their access, and kicked off an internal investigation for both mitigation and next steps.
+
+### Impact
+
+The promise of free access to Sourcegraph API prompted many to create accounts and start using the proxy app. The app and instructions on how to use it quickly made its way across the web,  generating close to 2 million views.
+
+As more users discovered the proxy app, they created free Sourcegraph.com accounts, adding their access tokens, and accessing Sourcegraph APIs illegitimately.
+
+The impact of the malicious user having admin access was limited to a subset of:
+
+- **Paid Customers**
+  - The license key recipient’s name and email address
+  - Sourcegraph license key
+
+<br/>
+
+- **Free-Tier Community Users**
+  - Email addresses
+
+<br/>
+
+We have no indication that any of this data was viewed, modified, or copied, but the malicious user could have viewed license key recipient’s emails and community user email addresses as they navigated the admin dashboard.
+
+**Important Note: Customers’ private data or code was not viewed during this incident. Customer private data and code resides in isolated environments and were therefore not impacted by this event.**
+
+### How we're mitigating
+
+As soon as we understood the scope of the incident we took the following steps:
+
+- Identified the malicious account and fully revoked its access
+- Proactively rotated a subset of Sourcegraph customer license keys that may have been viewed
+- Temporarily reduced the rate limits for all free community users
+- Created new processes and tests and will continue to monitor for malicious activity and abuse
+
+Expanding our secret scanning through additional static analysis tests will ensure we can better detect and prevent this kind of leak in the future.
```

Comment:
This feels a bit weird straight after the bullets - make it clear we will do this.